qiskit/pulse/analysis.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
```

Comment:
Follow the format in `transpiler`: these should be in a `passes/` dir, not top level `pulse`

Reply:
Sure, I can do this if we proceed. I just wanted to do it this way for backwards compatibility with `transforms` to start.